### DISCUSSION TITLE
Notepad++ UDL for 6502

### DISCUSSION THREAD

**Discussion by Pokeman2003** [vhelin/wla-dx#546] [Show and tell]
I'm not terribly sure of how useful this will be, but I wrote it anyways because I thought it was cool.  It supports all of the compatible WLA-DX preprocessors, and has all of the legal 6502 opcodes. [You can get it here.](https://pastebin.com/yRYL5j2c)
![image](https://user-images.githubusercontent.com/83632748/202585936-a3c3d082-03ab-418a-8e0e-6a5c4b853bb0.png)
If I could, I would've supported // comments as well, but unfortunately UDL2 files don't support that. Maybe someday in the future.

**Comment by milkeybabes** [comment]
<Keywords name="Keywords1">lda ldx ldy sta stx sty stz tax tay txa tya tsx txs txy tyx dex dey inx iny inc dec adc sbc cmp cpx cpy and eor ora bit asl lsr rol ror</Keywords>
<Keywords name="Keywords2">jmp jsr rts rti brk cop rti rtl rts bcc bcs beq bmi bne bpl bra brl bvc bvs pea pei per phk php pha phx phy pla plp plx ply phb phd plb pld</Keywords>
<Keywords name="Keywords3">clc sec cli sei cld sed clv rep sep xce wai stp nop mvn mvp jml jsl rtl tcd tdc tcs tsc txy tyx</Keywords>
<Keywords name="Keywords4">trb tsb rmb0 rmb1 rmb2 rmb3 rmb4 rmb5 rmb6 rmb7 smb0 smb1 smb2 smb3 smb4 smb5 smb6 smb7 bbr0 bbr1 bbr2 bbr3 bbr4 bbr5 bbr6 bbr7 bbs0 bbs1 bbs2 bbs3 bbs4 bbs5 bbs6 bbs7</Keywords>
<Keywords name="Keywords5">asl a lsr a rol a ror a</Keywords>


You can add the rest of the 65816 if anyone else finds this! Good old ChatGPT works wonders sometimes!

**Reply by Pokeman2003** [reply]
I might do it, but I'm quite busy at the moment.
